app/controllers/catalog_controller.rb

```diff
@@ -242,7 +242,7 @@ def atomic_form_field_changed
         if params[:st_prov_type] || (params[:display] && @edit[:new][:st_prov_type].starts_with?("generic"))
           page.replace("form_div", :partial => "st_form")
         end
-        if params[:display] || params[:template_id] || params[:manager_id]
+        if params[:display] || params[:template_id] || params[:manager_id] || params[:ovf_template_id]
```

Comment:
would be nice if this complex if statement (used twice in this file) can be extracted out into a tiny method named so that a developer can better understand the intent.  So, instead of `if params[:display] || params[:template_id] || params[:manager_id] || params[:ovf_template_id]`, it would say something like `if automate_catalog_needed?`
and that method would be defined as
```
def automate_catalog_needed?
  params[:display] || params[:template_id] || params[:manager_id] || params[:ovf_template_id]
end
```